### PR TITLE
June 2020 Updates

### PR DESCRIPTION
**Pull request by timahenning**
Addresses the June 2020 updates as well as Issue #21.

### CODE REVIEW

File: Hands-on lab/Before the HOL - Intelligent analytics.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Synopsis: In this exercise, you will set up your environment for use in the rest
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Before the HOL - Intelligent analytics.md:76]: Please update lines 30, 31, 51 & 109 from Task:  to Task 1 & Task 2.

File: README.md [hunk 2]
```diff
@@ -41,6 +41,8 @@ By the end of the hands-on lab, you will be more confident in the various servic
 - Azure Service Bus
```
Comment (DawnmarieDesJardins) [inline README.md:41]: Check products/services list...is Azure Logic App to be removed?

Comment (timahenning) [reply README.md:41]: Yes, this should be removed.

File: Whiteboard design session/WDS student guide - Intelligent analytics.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ Whiteboard design session student guide
 </div>
 
 <div class="MCWHeader3">
-March 2020
+June 2020
 </div>
```
Comment (DawnmarieDesJardins) [inline Whiteboard design session/WDS student guide - Intelligent analytics.md:14]: Logic Apps is in the additional references table.  Remove?

File: Whiteboard design session/WDS trainer guide - Intelligent analytics.md [hunk 4]
```diff
@@ -27,30 +27,30 @@ Microsoft and the trademarks listed at <https://www.microsoft.com/legal/intellec
...
+  - [Step 1: Review the customer case study](#step-1-review-the-customer-case-study-1)
+  - [Step 2: Design a proof of concept solution](#step-2-design-a-proof-of-concept-solution-1)
+  - [Step 3: Present the solution](#step-3-present-the-solution-1)
+  - [Wrap-up](#wrap-up-1)
+  - [Preferred target audience](#preferred-target-audience)
+  - [Preferred solution](#preferred-solution)
+  - [Checklist of preferred objection handling](#checklist-of-preferred-objection-handling)
+  - [Customer quote (to be read back to the attendees at the end)](#customer-quote-to-be-read-back-to-the-attendees-at-the-end)
 
 <!-- /TOC -->
```
Comment (DawnmarieDesJardins) [inline Whiteboard design session/WDS trainer guide - Intelligent analytics.md:56]: Logic Apps in the additional information chart.  Remove?

File: Hands-on lab/HOL step-by-step - Intelligent analytics.md [hunk 5]
```diff
@@ -387,9 +374,11 @@ In this task, you will create a new Event Hubs namespace and instance.
 
     - Select **Create**.
 
-      ![The Create Event Hub blade fields display with the previously mentioned settings.](media/image41.png 'Create Event Hub blade')
+      ![The Create Event Hub blade fields display with the previously mentioned settings.](media/2020-06-29-13-44-07.png "Create Event Hub blade")
+
+6. Repeat steps listed 5 to create another Event Hub. This one will store messages for archival and be processed by Stream Analytics. Stream Analytics forwards the message to Cogntive Search.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Intelligent analytics.md:379]: Cogntive Search (typo)

File: Hands-on lab/HOL step-by-step - Intelligent analytics.md [hunk 6]
```diff
@@ -1034,59 +947,39 @@ The namespace, and therefore connection string, for the service bus is different
...
+4. Create a new application setting with the **Name** `TextAnalyticsAccountKey` and paste the value of **Key 1** of the Cognitive Service into the **Value** field.
 
 5. Scroll to the top of **Application Settings** and select **Save** from the toolbar. Your application settings should now resemble the following (to see the hidden values, select the eye icon in the Value column):
 
-    ![The Application Settings listing for the Function App is shown.](media/2019-11-24-09-48-15.png "Function Application settings")
+    ![The Application Settings listing for the Function App is shown.](media/2020-06-30-19-52-53.png "Function Application settings")
 
 ## Exercise 3: Configure the Chat Web App settings
 
 Duration: 10 minutes
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Intelligent analytics.md:960]: Exercise 3 has no tasks/steps.

File: Hands-on lab/HOL step-by-step - Intelligent analytics.md [hunk 7]
```diff
@@ -2130,12 +1747,8 @@ In this exercise, attendees will deprovision any Azure resources that were creat
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Intelligent analytics.md:1747]: Line 1738 - Duration: 10 minutes (not mins please)

File: Whiteboard design session/WDS student guide - Intelligent analytics.md (hunk 3 above)
Comment (timahenning) [reply Whiteboard design session/WDS student guide - Intelligent analytics.md:14]: Fixed

**Pull request merged**